Common/Orders/OrderTypes.cs

```diff
@@ -68,6 +68,12 @@ public enum OrderDuration
         /// </summary>
         Day
         */
+
+        /// <summary>
+        /// Order valid until specific type.
+        /// </summary>
+        Specific
```

Comment:
Agree is API convenience and each order should carry a time where it expires (DateTime.MaxValue). Perhaps better description of this order duration would be "Custom"